src/view_graph.cpp

```diff
@@ -224,14 +224,16 @@ void View::DrawSmuflLine(
 {
     assert(dc);
 
-    int startWidth = (start == 0) ? 0 : m_doc->GetGlyphAdvX(start, staffSize, dimin);
+    if (length <= 0) return;
+
+    const int startWidth = (start == 0) ? 0 : m_doc->GetGlyphAdvX(start, staffSize, dimin);
+    const int endWidth = (end == 0) ? 0 : m_doc->GetGlyphAdvX(end, staffSize, dimin);
     int fillWidth = m_doc->GetGlyphAdvX(fill, staffSize, dimin);
-    int endWidth = (end == 0) ? 0 : m_doc->GetGlyphAdvX(end, staffSize, dimin);
 
-    if (length <= 0) return;
+    if (fillWidth == 0) fillWidth = m_doc->GetGlyphWidth(fill, staffSize, dimin);
```

Comment:
Similar to #1438. If the fill glyph has no `@h-a-x` Verovio crashes due to division by zero. This is to prevent these cases.